scripts/build.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
I really think it should be part of travis.yml because it is needed *only* there